Week2/homework/js-exercises/2-the-even-odd-reporter.js

```diff
@@ -0,0 +1,14 @@
+// Exercise 2: The even/odd reporter
+
+{
+    'use strict';
+    
+    for (let i = 0; i <= 20; i++) {
+        if ((i % 2) == 0) {
```

Comment:
Ah, I would highly recommend using triple equals operator here instead of double equals.